src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/ListingOption.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
```

Comment:
We already have State enum with these values. Can't that be used here?

Reply:
sure